configs/agent/sample.toml

```diff
@@ -267,6 +267,10 @@ ca-certs = ""
 keyfile = ""
 certfile = ""
 
+# If you want the source field in Graylog to display a name that distinguishes between components rather than the hostname, uncomment the lines below and insert a value into localname.
+# fqdn = false
+# localname = ""
```

Comment:
It would be better to set any non-empty string as a default value, as `localname` does not allow blank.
```suggestion
# localname = "agent"
```